src/code/z_collision_check.c

```diff
@@ -2998,7 +2998,7 @@ void CollisionCheck_ApplyDamage(GlobalContext* globalCtx, CollisionCheckContext*
         s32 i;
         u32 flags = info->acHitInfo->toucher.dmgFlags;
 
-        for (i = 0; i < 0x20; i++, flags >>= 1) {
+        for (i = 0; i < ARRAY_COUNT(tbl->table); i++, flags >>= 1) {
```

Comment:
idk if `ARRAY_COUNT` is okay here, because the table size is limited by the amount of bits in the `u32 flags` anyway. I feel like it may give a false sense of security about expanding it to use array_count but maybe I worry too much